iOS/ViewControllers/Courses/LTIExerciseHintViewController.swift

```diff
@@ -56,6 +62,14 @@ class LTIExerciseHintViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        self.itemTitleLabel.text = self.courseItem?.title
+        self.launchExerciseView.isHidden = true
+        self.exerciseTypeView.isHidden = true
+        self.pointsView.isHidden = true
```

Comment:
No need to hide views (like this one) if you "un-hide" them in `updateView()` immediately afterward. Hiding views here is only required for views at present data of `CourseItem.content`.